src/node_shell_read.js

```diff
@@ -33,10 +37,9 @@ readAsync = function readAsync(filename, onload, onerror) {
     onload(ret);
   }
 #endif
-  if (!nodeFS) nodeFS = require('fs');
-  if (!nodePath) nodePath = require('path');
+  requireNodeFS();
   filename = nodePath['normalize'](filename);
-  nodeFS['readFile'](filename, function(err, data) {
+  fs.readFile(filename, function(err, data) {
```

Comment:
Are you sure the quoting is not needed here? Is closure aware of the Node.js FS API?